v4/weaviate/graphql/hybridbuilder.go

```diff
@@ -0,0 +1,40 @@
+package graphql
+
+import (
+	"fmt"
+	"strings"
+)
+
+type HybridArgumentBuilder struct {
+	Query  string
+	Vector []float32
+	Alpha  float32
+}
+
+// WithQuery the search string
+func (e *HybridArgumentBuilder) WithQuery(query string) *HybridArgumentBuilder {
+	e.Query = query
+	return e
+}
+
+// WithVector the vector.  Can be omitted
+func (e *HybridArgumentBuilder) WithVector(vector ...float32) *HybridArgumentBuilder {
```

Comment:
shouldn't a vector argument be declared as `vector []float32` but not in dot notation? (that would be consistent with other arguments)

Reply:
That sounds good